extensions/resource-deployment/src/ui/deployClusterWizard/pages/azureSettingsPage.ts

```diff
@@ -38,9 +38,7 @@ export class AzureSettingsPage extends WizardPageBase<DeployClusterWizard> {
 					description: localize('deployCluster.SubscriptionDescription', "The default subscription will be used if you leave this field blank.")
 				}, {
 					type: FieldType.ReadonlyText,
-					label: '',
-					labelWidth: '0px',
-					defaultValue: localize('deployCluster.SubscriptionHelpText', "{0}"),
+					label: localize('deployCluster.SubscriptionHelpText', "{0}"),
```

Comment:
The localize call here is unnecessary if it's just going to be a placeholder value

Reply:
It was prexising but will remove

Reply:
fixed.